perfume-api/src/main/java/io/perfume/api/sample/application/dto/SampleResult.java

```diff
@@ -0,0 +1,4 @@
+package io.perfume.api.sample.application.dto;
+
+public record SampleResult (Long id, String name) {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.MethodParamPadCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'(' is preceded with whitespace.